sdk/purview/azure-analytics-purview-sharing/src/samples/java/com/azure/analytics/purview/sharing/ReadmeSamples.java

```diff
@@ -0,0 +1,328 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
```

Comment:
Were your readme samples also autogenerated? If so, that's rad.

Reply:
These were not autogenerated.  I wrote these tests.